src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java

```diff
@@ -24,7 +24,7 @@ public class SshSlaveLauncher extends ComputerLauncher {
     public final Control suffixCmd = control("suffixStartSlaveCmd");
     public final Control timeout = control("launchTimeoutSeconds");
     public final Control retries = control("maxNumRetries");
-    public final Control javaPath = control("javaPath");
+    private final Control javaPath = control("javaPath");
```

Comment:
all the controlls should really be provate and hidden behind get/set but that is out of scope in this change